app/globals.css

```
@@ -0,0 +1,14 @@
+
+:root {
+  --background: #fafaf7;
+  --foreground: #1a1a1a;
+  --accent: #c2741f;
+}
+
+html,
+body {
+  background: var(--background);
+  color: var(--foreground);
+  font-family: "DM Sans", "Plus Jakarta Sans", "Nunito", system-ui, sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Nunito" (font-family-name-quotes)

(font-family-name-quotes)